DSL.Resql/list_entity.sql

```diff
@@ -0,0 +1 @@
+select * from Entity;
```

Comment:
Avoid `select *` in any case and be explicit on what you are requesting.

Reply:
Done